### PR TITLE
doc(node): add `KernelModulesLoaded` node condition

### PR DESCRIPTION
**Pull request by mantissahz**
#### Which issue(s) this PR fixes:

Issue # longhorn/longhorn#9153

### CODE REVIEW

File: content/docs/1.7.2/nodes-and-volumes/nodes/node-conditions.md [hunk 1]
```diff
@@ -25,5 +25,9 @@ Node conditions:
 - `NFSClientInstalled`:  
   Identifies if any of the following NFS clients are supported: `v4.2`, `v4.1`, or `v4.0`. NFS client is required for RWX volume and backup.  
 
+- `KernelModulesLoaded`:  
+  Identifies if the following Kernel modules are loaded:
+  - `dm_crypt`: It is required for the volume and backing image encryption.
```
Comment (jhkrug) [inline content/docs/1.7.2/nodes-and-volumes/nodes/node-conditions.md:30]:
```suggestion
  - `dm_crypt`: Is required for the volume and backing image encryption.  
```

File: content/docs/1.8.0/nodes-and-volumes/nodes/node-conditions.md [hunk 2]
```diff
@@ -25,5 +25,9 @@ Node conditions:
 - `NFSClientInstalled`:  
   Identifies if any of the following NFS clients are supported: `v4.2`, `v4.1`, or `v4.0`. NFS client is required for RWX volume and backup.  
 
+- `KernelModulesLoaded`:  
+  Identifies if the following Kernel modules are loaded:
+  - `dm_crypt`: It is required for the volume and backing image encryption.
```
Comment (jhkrug) [inline content/docs/1.8.0/nodes-and-volumes/nodes/node-conditions.md:30]:
```suggestion
  - `dm_crypt`: Is required for the volume and backing image encryption.  
```

**Review by jhkrug — CHANGES_REQUESTED** [review]
A very minor suggestion for your consideration.

File: content/docs/1.7.2/nodes-and-volumes/nodes/node-conditions.md (hunk 1 above)
Comment (mantissahz) [reply content/docs/1.7.2/nodes-and-volumes/nodes/node-conditions.md]: Modified. Thanks.

File: content/docs/1.8.0/nodes-and-volumes/nodes/node-conditions.md (hunk 2 above)
Comment (mantissahz) [reply content/docs/1.8.0/nodes-and-volumes/nodes/node-conditions.md]: Modified. Thanks.

**Review by jhkrug — APPROVED** [review]
lgtm

**Pull request merged**